src/services/export.js

```diff
@@ -224,17 +251,15 @@ const saveEpub = async page => {
   // screenshot replacements have to come after image src changes
 
   // replace gadgets
-  const gadgets = await page.$$('div.gadget-content');
-  const gadgetScreenshots = await screenshotElements(gadgets, page);
-  await replaceElementsWithScreenshots(gadgets, page);
+  const gadgets = await page.$$('div.gadget');
+  const gadgetScreenshots = await replaceElements(gadgets, page, interactive);
 
   // remove panels accompanying gadgets
-  await page.$$eval('div.panel', els => els.forEach(el => el.remove()));
+  // await page.$$eval('div.panel', els => els.forEach(el => el.remove()));
 
   // replace embedded html divs, including youtube videos
-  const embeds = await page.$$('div.embedded-html');
-  const embedScreenshots = await screenshotElements(embeds, page);
-  await replaceElementsWithScreenshots(embeds, page);
+  const embeds = await page.$$('.resources object');
+  const embedScreenshots = await replaceElements(embeds, page, interactive);
```

Comment:
I think that the `if` statement for `interactive` should come here. It's a bit weird to be calling the `replaceElements` method with the `interactive` flag, if this flag means that nothing will be replaced. I think it's better to do the `if` here:

```
let embedScreenshots = null;
if (interactive) {
  await adjustHeightForElements(elements, page); 
} else {
  embedScreenshots = await ...
}
```

This is also better because in one case you need the embedScreenshots, in the other, you just leave it as `null`.